code/go/0chain.net/smartcontract/rest/rest.go

```diff
@@ -0,0 +1,53 @@
+package rest
+
+import (
+	"0chain.net/chaincore/chain/state"
+	"net/http"
+)
+
+type RestEndpoint struct {
```

Comment:
Since the package name is rest, so we can rename this struct to `Endpoint` to reduce repetition.